src/rules/unbound-method.ts

```diff
@@ -0,0 +1,101 @@
+import { TSESLint, TSESTree } from '@typescript-eslint/experimental-utils';
+import { createRule, isExpectCall, parseExpectCall } from './utils';
+
+const toThrowMatchers = [
+  'toThrow',
+  'toThrowError',
+  'toThrowErrorMatchingSnapshot',
+  'toThrowErrorMatchingInlineSnapshot',
+];
+
+const isJestExpectCall = (node: TSESTree.CallExpression) => {
+  if (!isExpectCall(node)) {
+    return false;
+  }
+
+  const { matcher } = parseExpectCall(node);
+
+  return !toThrowMatchers.includes(matcher?.name ?? '');
```

Comment:
`'foobar'.includes('')` is `true`. Can we get the name earlier then bail rather than calling `includes`?

Reply:
I agree - do you have a preference on `!matcher ||` vs `if`?

Reply:
```
if (!matcher?.name) {
  // or true, whatever
  return false;
}

return !toThrowMatchers.includes(matcher.name);